purchases-sample/src/androidTest/java/IntegrationTest.kt

```diff
@@ -0,0 +1,43 @@
+
+import androidx.test.ext.junit.rules.activityScenarioRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.revenuecat.purchases.Purchases
+import com.revenuecat.purchases.getPurchaserInfoWith
+import com.revenuecat.sample.InitialActivity
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.fail
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.util.concurrent.CountDownLatch
+import java.util.concurrent.TimeUnit
+
+@RunWith(AndroidJUnit4::class)
+class IntegrationTest {
```

Comment:
I'm not a huge fan of having the integration tests be a part of the sample app...
the sample app is meant for customers who want to see how to use RevenueCat, and it should contain only stuff that they could actually use, and it should evolve over time into a pretty big thing that customers can grab chunks off of for copy/pasting. 
the integration tests have an entirely different set of concerns and constraints, I think they're merit being separated, even if for now they're small